pkg/cli/mapping_server.go

```diff
@@ -120,7 +120,14 @@ func (c *MappingServerCommand) RunUnstarted(ctx context.Context, args []string)
 	}
 	closer = multicloser.Append(closer, assetClient.Close)
 
-	processor, err := processors.NewAssetInventoryProcessor(ctx, assetClient, fmt.Sprintf("projects/%s", c.cfg.ProjectID))
+	var defaultScope string
+	if c.cfg.ResourceScope == "" {
+		defaultScope = fmt.Sprintf("projects/%s", c.cfg.ProjectID)
```

Comment:
This project ID is used when we creating the pubsub client for sucess and failure messengers which is the pmap project. I don't think we should use this as default for searching cloud resource assets, an option is to make DefaultResourceScope required.

Reply:
The current behavior was defaulting to scope to the current project using the project ID, I just want to keep that behavior.

Reply:
That's too implicit, we can make it more explicit by making support GCS configurable, when it is on (support GCS), default resource scope is required. Another easier option is to make default resource scope required for all cases, since our customer can easily use their gcp org as the default scope.

Reply:
Made resource scope a required value.